test/js/node/missing-module.test.js

```diff
@@ -0,0 +1,7 @@
+import { expect, test } from "bun:test";
+
+test("not implemented yet module masquerades as undefined and throws an error", () => {
```

Comment:
it doesn't masqsuerade as undefined, masquerade as undefined is a function which is callable but typeof returns `"undefined"`